### DISCUSSION TITLE
To switch back to normal mode automatically after inaction

### DISCUSSION THREAD

**Discussion by CrCati** [JetBrains/ideavim#585] [Ideas]
That will be very cool to add a VIM feature "To switch back to normal mode automatically after inaction.".

Maybe Just like this:
https://vim.fandom.com/wiki/To_switch_back_to_normal_mode_automatically_after_inaction

**Comment by AlexPl292** [comment]
Hi there, you can track this issue here: https://youtrack.jetbrains.com/issue/VIM-2043/how-to-setup-exit-from-insert-mode-after-some-time
